docs/development/setup-advanced.md

```diff
@@ -129,6 +148,21 @@ installation method described here.
 
 1. [Visual Studio Code Remote - WSL](https://code.visualstudio.com/docs/remote/wsl) is
    recommended for editing files when developing with WSL.
+   From the zulip directory run:
+
+   ```bash
+   code .
+   ```
+
+   Install the appropriate extensions in VS Code, such as:
+
+   - Remote Development Extension Pack
+   - Languages in use
+   - ESLint
+   - Prettier
+   - OpenAPI(Swagger)
+   - Docker
+   - Jupyter
```

Comment:
I think this list is best expressed in code, in the form of a `.vscode/extensions.json` file.

Except the first item, perhaps? That one looks like it may be specific to this environment. That'd be good to highlight, then. A link to the specific extension (and/or a mention of its unique name -- like how the ESLint extension is `dbaeumer.vscode-eslint`) would also be good, so it's easy to be sure one's installing the same extension we meant.

Reply:
Lauryn just created this file, so I can add to it if that is the best way.  These extensions are already in the file, except for the remote development pack (`ms-vscode-remote.vscode-remote-extensionpack`).  Would it be better to put that one here, because it only applies to this, or add it to the `.vscode/extensions.json` file?
I'll draft it with the extension here, but it will be easy enough to change if it seems better the other way.

Reply:
If it only applies to WSL, then best to leave it here. Perhaps also add it as a comment in the extensions.json file?

I wonder, though, if it might be something that many other VS Code users would find helpful with the Vagrant development environment too, or with [remote development servers](https://zulip.readthedocs.io/en/latest/development/remote.html). It [looks like](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) it's designed for that use case as well as for WSL. Could be a good thing to bring up on perhaps #general -- perhaps other people will have experience to report from using it.

(No need to wait for a conclusion there before recommending it for WSL here, though.)